simple/stats/config.md

```diff
@@ -92,6 +120,32 @@ and this field associates one of the provenances defined there to this file.
 Provenances typically map to a dataset from a source.
 e.g. WorldDevelopmentIndicators provenance (or dataset) is from the WorldBank source.
 
+#### `rowEntityType`
+
+For entity imports, every row represents an entity.
+The type of entity that each row represents can be specified using `rowEntityType`.
+This is a mandatory field for entity imports.
+
+#### `idColumn`
+
+Applicable to events and entities.
+
+Every event and entity is identified by a unique ID.
+If one of the columns encodes that ID, it can be indicated with the `idColumn` field.
+If none is specified, IDs will be auto generated.
+
+#### `entityColumns`
+
+Applicable to entities.
```

Comment:
what if this is not specified?

Reply:
Then all columns will be considered object values.